source/particles/particle_handler.cc

```diff
@@ -1025,7 +1025,7 @@ namespace Particles
   template <int dim, int spacedim>
   void
   ParticleHandler<dim, spacedim>::register_store_callback_function(
-    const bool serialization)
+    const bool /*serialization*/)
```

Comment:
Can we just remove this argument altogether?

Reply:
I don't know if this would affect ASPECT. Shall we deprecate it first? @gassmoeller -- FYI

Otherwise, I would agree to remove it.

Reply:
Let's just do it. We can easily adjust ASPECT.